kite_test.go

```diff
@@ -263,3 +271,9 @@ func SquareCB(r *Request) (interface{}, error) {
 
 	return nil, nil
 }
+
+func NewKite(name, version string) *Kite {
```

Comment:
NewXHRPollingKite may be? yes it is a long one but more expressive :) On the other hand if it is just a test helper you can un-export it